sistr/tests/test_cgmlst.py

```diff
@@ -0,0 +1,39 @@
+'''
```

Comment:
Also, I'm wondering how you were running the tests (or if you were making use of them)? I tried running `pytest` but I get a number of errors like:

```
ImportError while importing test module '/home/CSCScience.ca/apetkau/workspace/sistr_cmd/sistr/tests/test_hclust_cutree.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
sistr/tests/test_hclust_cutree.py:1: in <module>
    from sistr.src.cgmlst.extras.hclust_cutree import profiles_to_np_array, dist_matrix_hamming, complete_linkage, cutree, nr_profiles, expand_clusters_dataframe
sistr/src/cgmlst/extras/hclust_cutree.py:1: in <module>
    from fastcluster import linkage
E   ModuleNotFoundError: No module named 'fastcluster'

sistr/tests/test_writer_util.py:5
  /home/CSCScience.ca/apetkau/workspace/sistr_cmd/sistr/tests/test_writer_util.py:5: PytestCollectionWarning: cannot collect test class 'Test1' because it has a __init__ constructor
    class Test1:
```

Reply:
I have removed the duplicated test directory. At present I am not using the tests, they were designed by Peter. I think that we can add for a future issue adding full coverage of the SISTR code base by tests.

Reply:
Thanks. That sounds good to me. Could you make that issue on github (to add tests for SISTR code base)?